README.md

```diff
@@ -1,38 +1,38 @@
 # Dasynq
 
-Dasynq is an event loop library similar to libevent, libev and libuv. Like other such libraries, it is
-crossplatform / portable. Unlike most other such libraries, it is intended to be completely usable in
-a multi-threaded client program, and it is written in C++; furthermore the API is designed to allow
+Dasynq is an event loop library similar to libevent, libev and libuv. Similar to other libraries, it is
```

Comment:
"such" is not redundant here, because the comment is about event loop libraries, not about libraries in general.